src/styles/ngx-bootstrap/_cards.scss

```diff
@@ -1,11 +1,6 @@
 @import '../globals';
 @import '../colors';
 
-$card-border-color: map-get($color-background, 'darker');
-$card-border-radius: $border-radius;
-
 .card {
-	border-color: $card-border-color;
```

Comment:
@jrassa do you still need to include this use of the SCSS variable in the class definition, or is that handled through some other mechanism?